### ISSUE TITLE
Organizations screen doesn't work on localhost

### ISSUE THREAD

**Issue by aceat64** [p2-inc/keycloak-ui#7]
I'm running `quay.io/phasetwo/phasetwo-keycloak:19.0.3-alpha` on my local machine with Docker. When clicking on Organizations the page fails to load and displays the following error:

> NetworkError when attempting to fetch resource.
<img width="917" alt="Screenshot 2022-10-10 185848" src="https://user-images.githubusercontent.com/348675/194969642-b7bce402-2a88-4235-8072-eef9bfbc3206.png">

The browser console shows the following:

![image](https://user-images.githubusercontent.com/348675/194969737-a46523aa-8fd5-42d9-8dac-976c81f10eba.png)

The generated file `index.e3df50dc.js` has only one instance with the `app.phasetwo.io/auth` URL:

``` javascript
AUTH_URL=window.location.href.includes("localhost")?"https://app.phasetwo.io/auth":"/auth"
```

I briefly searched through this repo to see if I could find what was causing these requests to go to the hard-coded URL, but wasn't able to determine the cause. Perhaps a build setting?

Steps to reproduce:

1. Start container
    ``` shell
    docker run -p 8080:8080 -e KEYCLOAK_ADMIN=admin -e KEYCLOAK_ADMIN_PASSWORD=admin -e KC_HTTP_RELATIVE_PATH=/auth  quay.io/phasetwo/phasetwo-keycloak:19.0.3-alpha start-dev
    ```
2. Login to <http://localhost:8080/auth>
3. In Realm Settings > Theme: set "Admin console theme" to "phasetwo.v2"
4. Refresh page
5. Click on Organizations

Accessing the admin console by IP address (e.g. `http://127.0.0.1:8080/auth`) works fine.

**Comment by xgp** [comment]
Looks like a bug when running on localhost. Try using an IP or other name for now.

**Comment by tomschulze** [comment]
I experience the same issue.

@aceat64 I think you already pasted the line of code which causes the bug. The `baseUrl` for the [orgFetcher](https://github.com/p2-inc/keycloak-ui/issues/8#issuecomment-1301256855) seems to be set to `https://app.phasetwo.io/auth` if `window.location.href` includes `localhost`; which also seems to be the reason for the CORS error in the browser console.

https://github.com/p2-inc/keycloak-ui/blob/19.0.3_orgs/apps/admin-ui/src/environment.ts#L25

**Comment by xgp** [comment]
fixed in 20.0.1_orgs
